src/main/java/net/rpgtoolkit/common/utilities/TileSetCache.java

```diff
@@ -20,7 +27,7 @@
 public class TileSetCache {
 
   // Singleton.
-  private static final TileSetCache instance = new TileSetCache();
+  private static final TileSetCache INSTANCE = new TileSetCache();

```

Comment:
You mean like change all its members and methods to static? 

I dunno when I wrote this class I just chose the Singleton pattern by default. Making it static won't have much impact really, right?